### ISSUE TITLE
Cold Hearted

### ISSUE THREAD

**Issue by styled81** [dalaranwow/dalaran-wow#5074]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: The dragons fly terribly slow, realy slow... 1st drake did almost go as intended. first few sec it was very slow, when i shot a another captive protodrake the speed of the dragon was as good. 2nd dragon was from the start slow and never did go any faster. Also impossible to dive/climb with the 2nd dragon, 1 static level of flight is possible.

**Expected behaviour**: Speed of the dragon isnt right and i should steer the protodragon.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or 

https://www.youtube.com/watch?v=WcdC3TvfSe8

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3904
